package.json

```diff
@@ -100,19 +101,13 @@
     "eslint": "^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8"
   },
   "dependencies": {
-    "array-includes": "^3.1.6",
-    "array.prototype.flat": "^1.3.1",
-    "array.prototype.flatmap": "^1.3.1",
     "debug": "^3.2.7",
     "doctrine": "^2.1.0",
     "eslint-import-resolver-node": "^0.3.7",
     "eslint-module-utils": "^2.8.0",
     "get-tsconfig": "^4.6.0",
-    "has": "^1.0.3",
-    "is-core-module": "^2.12.1",
     "is-glob": "^4.0.3",
     "minimatch": "^3.1.2",
-    "object.values": "^1.1.6",
     "resolve": "^1.22.3",
     "semver": "^6.3.0"
```

Comment:
> I have bumped the semver to the latest version in the latest commit, which basically fixes all these issues and PRs:

@SukkaW 

I didn't see this changed?

Reply:
The commit must be missed during rebase! I just re-push it: 20a0858ea7c6a71b931547d1f4e72441bc8596a0